core-concepts/wallets/veworld/user-guide/wallet.md

```diff
@@ -0,0 +1,244 @@
+---
+description: How to create, import and export wallets on VeWorld
+layout:
+  title:
+    visible: true
+  description:
+    visible: true
+  tableOfContents:
+    visible: true
+  outline:
+    visible: true
+  pagination:
+    visible: true
+---
+
+# Wallet
+
+## Create a new wallet
+
```

Comment:
I think a few words' introduction to the two wallets makes sense here. But maybe you have it already before.
Nothing too complicated, "VeWorld is available as a mobile wallet and as a browser extension. The process to create a wallet is similar, the following sections will cover the differences".

Reply:
Good point that would require a broader conversation... 
We should define the level of knowledge that we expect from a user who read this guide.